pages/[lng]/article/[slug].tsx

```diff
@@ -62,9 +67,13 @@ const ArticleDetail: FC<any> = ({
 
 export const getServerSideProps: GetServerSideProps = async ({
   req,
+  res,
   params,
 }) => {
-  const brand = await useBrand(req)
+  const [brand, ] = await Promise.all([
+    useBrand(req),
+    useAuthToken({ req, res, env: process.env }),
+  ]);
```

Comment:
```suggestion
  ])
```
titik koma-nya dihapus pacc biar seragam
mungkin kalo ada yang di file lain juga sama dibegitukan